src/lib/php/UI/template/components/login-form.html.twig

```diff
@@ -24,10 +24,26 @@
           <input id="unamein" type="text" class="form-control" name="username" placeholder="Enter username" autofocus="autofocus" value="{{ userName|e }}">
         </div>
       </div>
+      <meta charset="UTF-8" />
+      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      <title>Toggle Password Visibility</title>
+      <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css" />
+      <link rel="stylesheet" href="css/style.css" />
       <div class="form-group">
-        <label class="control-label col-sm-2" for="pwd">{{ 'Password'|trans }}:</label>
+      <label class="control-label col-sm-2" for="pwd">{{ 'Password'|trans }}:</label>
         <div class="col-sm-10">
-          <input type="password" class="form-control" id="pwd" placeholder="Enter password" name="password">
+          <input type="password" id="pwd" placeholder="Enter password" name="password" style="display: initial;
+          width: 100%;
+          padding: 0.375rem 0.75rem;
+          font-size: 1rem;
+          line-height: 1.5;
+          color: #495057;
+          background-color: #fff;
+          background-clip: padding-box;
+          border: 1px solid #ced4da;
+          border-radius: 0.25rem;
+          transition: border-color .15s ease-in-out,box-shadow .15s ease-in-out"/>
+          <i class="bi bi-eye-slash" id="togglePassword" style="margin-left: -30px; cursor: pointer;"></i>
```

Comment:
Can something like [button addons](https://getbootstrap.com/docs/4.0/components/input-group/#button-addons) be used instead?

Otherwise, please use relative lengths instead of absolute lengths like `px`.

Reply:
okay will update the changes